### DISCUSSION TITLE
Mock Receipt

### DISCUSSION THREAD

**Discussion by andrewl94** [imdhemy/laravel-in-app-purchases#116] [Q&A] [answered]
Hi! On online Documentation there is this Mock Receipt, but I do not find any documentation or example to do such:
````
Mock Receipts
You can start developing your back-end functionality without real receipts at all. App store IAP gives you an easy way to mock all responses from the App store.
````
url: https://imdhemy.com/laravel-iap-docs/docs/appstore-iap/intro#mock-receipts

Can anyone provide the link or example to test receipts this way?

**Comment by imdhemy** [comment, marked as answer]
Sorry for the late reply.

Here is a guide on [how to mock receipts for App Store](https://imdhemy.com/laravel-iap-docs/docs/app-store/testing-and-mocking), and Google play works the same.
